### ISSUE TITLE
Update PyGRB Workflow Generation

### ISSUE THREAD

**Issue by jakeb245** [gwastro/pycbc#4045]
Lots of work has been done on the PyGRB pipeline, but the workflow generation script hasn't been updated in 3 years! It is quite out of date with rest of the pipeline and PyCBC as a whole. Work is being done to bring the workflow generation up to date in [this branch](https://github.com/jakeb245/pycbc/tree/wflow-test1). Any discussion, advice, etc. is welcome!

Things done so far:
- Updated segment generation to work with current PyCBC modules
- Updated workflow to successfully run pycbc_multi_inspiral (that is: a run with pycbc_hdf5_splitbank and pycbc_multi_inspiral as executables)
- Various fixes along the way to make the above things work

Things to do:
- Reimplement injections, time slides, and post-processing with updated code
- Either reimplement gating in workflow generation or move gating into workflow
- Keep workflow up to date with current development
- Move some LAL dependent files around in the GRB utilities code
- Fix bugs!

**Comment by ahnitz** [comment]
@jakeb245 Once this is complete, I might suggest adding a unittest similar to what we have for the coincident search workflow (i.e. a full run of some analysis but with settings to bring the runtime to ~ 30 minutes). That way we can prevent anyone breaking the pipeline and if changes are needed they have to be reflected there too.

**Comment by pannarale** [comment]
I'm closing this issue as it is superseded by https://github.com/gwastro/pycbc/pull/4288 (and future PRs that will follow it).
